### ISSUE TITLE
Can't play Skyrim SE and The Witcher 3 freezes

### ISSUE THREAD

**Issue by Naicriz** [italomandara/CXPatcher#58]
Sonoma latest version
CrossOver 22.1
Game Porting ToolKit 1.0.2 
CXPatcher 0.2.16

Tried cleans installs of crossover and cxpatcher. Installed steam from crossover, from a downloaded exe. The Witcher 3/SkyrimSE from steam, from crossover, etc... All has been reinstalled at least 10 times (including downloading again all the games).
Also i've tried installing the witcher 3 and skyrim on different bottles. restoring bottles, cleaning, etc...
God knows how many times i've tried hahaha, the problems are the following;

Skyrim SE 64bits;
- When launching the game, wine gives an error that something went wrong and can not open the launcher, so the game doesn't run.
- I Tried with the properties command on steam; `--launcher-skip` but i think the game needs the launcher or wont run.

The Witcher 3;
- Have the properties command on steam; `--launcher-skip`, because without it wont launch correctly.
- the game run smooth until i hit an entity or viceverse. When that happen the game freezes completely, i try to force quit or quit all applications on crossover but it does not close the apps.
- I use this command to terminate the process:
`killall -9 wineserver && killall -9 wine64-preloader
`

**Comment by italomandara** [comment]
I can test the witcher 3 and see what's going on, on the other hand can't test Skyrim, have you tried to patch a copy of crossover without enabling the gptk installation?

**Comment by Naicriz** [comment]
> I can test the witcher 3 and see what's going on, on the other hand can't test Skyrim, have you tried to patch a copy of crossover without enabling the gptk installation?

No, I have not tried without gptk patch. or maybe i did, because i was able to play the witcher 3 for a period of time without problems but after a hour or so it freezes as it happen before.

I'll try it without the gptk patch.

Also, i have to enable the "allow repatch/upgrade" if is a clean install or is i have it before? it doesn't affect in someway?

**Comment by Naicriz** [comment]
Update:
- I tried without patching crossover with GPTK and Skyrim SE run without problems i think. 
The witcher 3 i would like to use de dx12, so i haven't (also because i've installed it so many times, its a lot of time.)

**Comment by italomandara** [comment]
Can you try with gptk 1.0.3 by repatching your crossover app?
